lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -15,6 +15,32 @@ html.video-fullscreen {
   }
 }
 
+.problem-reset-btn {
```

Comment:
I believe this is not the desired styling…
![image](https://user-images.githubusercontent.com/3189670/133650809-980a5684-f9f0-4713-aa39-854ebcfe5929.png)


Anyway, if we decide to keep the styling, let's move it under [.vert](https://github.com/edx/edx-platform/blob/46252f896a1f3cecb9cb05694ef7186e259e16e7/lms/static/sass/course/courseware/_courseware.scss#L658).